### PR TITLE
feat: add author details

### LINKED ISSUE #1389: [Author Details] User info module
As per this binding Governance proposal voted by the community: https://governance.decentraland.org/proposal/?id=37a9f190-5a4d-11ee-81f5-ad28f86a7c4c we will be adding more contextual information to the user modal on active Grant proposals

- [x] The module will appear in active Grant proposals
- [x] Will display user avatar, username and account linkage (if any)
- [x] Will display info about the first time the user voted in the DAO
- [x] Will display a link to the user profile

If the user is a first-time grant requester

- [x] Will display a text stating that this is a first-time grant request
- [x] Will display the metric of 30-day participation.

Image

If the user has already requested a grant:

- [x] Will display the total amount of money granted for Grants (to discuss: what if the user got a grant revoked?)
- [x] Will display the amount of past grant requests the user made
- [x] Will display the status of past projects: Ongoing, Finished, Paused, Revoked
- [x] Will display the metric of 30-day participation.

Image

Nice to have

- [ ] The active badges that the user has

Image

 will be displayed below the username

### PR DESCRIPTION
**Pull request by andyesp**
**Screenshots**
image
image

**Related Issue**
Closes #1389 

- [ ]  **Functionality validation**
    - [ ]  Feature working as described in the related issue
    - [x]  New environmental variables have been set
    - [x]  New API services methods have tests
    - [x]  Code errors are being logged in the code health service
- [ ]  **Product validation**
    - [ ]  UI changes look the same as in Figma
    - [x]  Usage events are being sent to the product analytics service
    - [x]  New common components have been documented in Storybook
    - [x]  Preview link if it’s a customer facing issue
- [x] **Documentation**
    - [x] README updated
    - [x] New API changes are documented
- [ ] **Accessibility**
    - [ ] UI changes are accessible

### CODE REVIEW

**Review by ncomerci — APPROVED** [review]

File: src/components/Proposal/View/AuthorDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        return {
+          ...grant,
+          vesting_released: vesting?.vesting_released || 0,
+          vesting_status: vesting?.vesting_status,
+        }
+      }),
+    [vestings, grants?.data]
+  )
+  const fundsReleased = useMemo(
+    () => grantsWithVesting?.reduce((total, grant) => total + grant.vesting_released, 0),
```
Comment (1emu) [inline src/components/Proposal/View/AuthorDetails.tsx:51]: This memos could be refactored into one so we traverse the list just once

```typescript
const {
  fundsReleased,
  finishedProjectsLength,
  revokedProjectsLength,
  ongoingProjectsLength,
  pausedProjectsLength,
} = useMemo(() => {
  let fundsReleased = 0;
  let finishedProjectsLength = 0;
  let revokedProjectsLength = 0;
  let ongoingProjectsLength = 0;
  let pausedProjectsLength = 0;

  if (grantsWithVesting) {
    grantsWithVesting.forEach((grant) => {
      fundsReleased += grant.vesting_released;

      switch (grant.vesting_status) {
        case ProjectStatus.Finished:
          finishedProjectsLength += 1;
          break;
        case ProjectStatus.Revoked:
          revokedProjectsLength += 1;
          break;
        case ProjectStatus.InProgress:
          ongoingProjectsLength += 1;
          break;
        case ProjectStatus.Paused:
          pausedProjectsLength += 1;
          break;
        default:
          // Handle default case if needed
          break;
      }
    });
  }

  return {
    fundsReleased,
    finishedProjectsLength,
    revokedProjectsLength,
    ongoingProjectsLength,
    pausedProjectsLength
  };
}, [grantsWithVesting]);
```

File: src/hooks/useVestings.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import { DclData } from '../clients/DclData'
+
+import { DEFAULT_QUERY_STALE_TIME } from './constants'
+
+function useVestings(shouldFetch: boolean) {
+  const { data, isLoading: isLoadingVestingsData } = useQuery({
+    queryKey: ['vestings'],
+    queryFn: () => DclData.get().getVestings(),
+    staleTime: DEFAULT_QUERY_STALE_TIME,
```
Comment (1emu) [inline src/hooks/useVestings.ts:11]: I think this stale time could be 24hs, if the exported data from transparency only gets updated daily

**Review by 1emu — APPROVED** [review]
✨

File: src/components/Proposal/View/AuthorDetails.tsx (hunk 1 above)
Comment (andyesp) [reply src/components/Proposal/View/AuthorDetails.tsx:55]: Updated! Check it out!

**Review by 1emu — APPROVED** [review]

**Review by ncomerci — APPROVED** [review]

**Review by 1emu — APPROVED** [review]

**Pull request merged**